tests/test_worker.py

```diff
@@ -46,6 +46,15 @@ class CustomJob(Job):
 class CustomQueue(Queue):
     pass
 
+class CustomSerializer():
```

Comment:
Can we rename this to `JSONSerializer` and move this to `serializers.py`? We can then document this as the recommended way to use JSON as serialization method.

Reply:
Perfect, updated with this change. Should the edits to the doc be made in this PR?